bitnami/drupal/Chart.yaml

```diff
@@ -31,4 +31,4 @@ name: drupal
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/drupal
   - https://www.drupal.org/
-version: 12.5.3
\ No newline at end of file
+version: 12.5.4
```

Comment:
Could you please update the version? `12.5.4` has been released already: https://github.com/bitnami/charts/blob/master/bitnami/drupal/Chart.yaml